internal/venice-common/src/main/java/com/linkedin/venice/pubsub/adapter/kafka/consumer/ApacheKafkaConsumerAdapter.java

```diff
@@ -56,7 +56,7 @@ public class ApacheKafkaConsumerAdapter implements PubSubConsumerAdapter {
   private final PubSubMessageDeserializer pubSubMessageDeserializer;
   private final ApacheKafkaConsumerConfig config;
 
-  ApacheKafkaConsumerAdapter(
+  public ApacheKafkaConsumerAdapter(
```

Comment:
Shouldn't be public. See the comment about writing test against generic PubSubConsumerAdapter